### PR TITLE
Added 3.9.0

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

**Review by asimons-mule — APPROVED** [review]
Thanks! Can we just make it 3.9.x rather than 3.9.0 and 3.9.1+

File: modules/ROOT/pages/standalone-mule-setup.adoc [hunk 1]
```diff
@@ -8,6 +8,7 @@ endif::[]
 Anypoint Visualizer supports the following Mule versions for standalone Mule deployments:
 
 * Mule 3.8.7
+* Mule 3.9.0
```
Comment (sarathecoubian) [inline modules/ROOT/pages/standalone-mule-setup.adoc]:
```suggestion
```

File: modules/ROOT/pages/standalone-mule-setup.adoc [hunk 2]
```diff
@@ -8,6 +8,7 @@ endif::[]
...
 * Mule 3.9.1+
```
Comment (sarathecoubian) [inline modules/ROOT/pages/standalone-mule-setup.adoc]:
```suggestion
* Mule 3.9.x
```

**Review by mcherem — APPROVED** [review]
Peer review.

**Pull request merged**